rhombus-url/rhombus/net/scribblings/rhombus-url.scrbl

```diff
@@ -0,0 +1,212 @@
+#lang rhombus/scribble/manual
+@(import:
+    meta_label:
+      rhombus open
+      net/url open)
+
+@title(~style: #'toc){Rhombus URL and Form Parsing}
+
+@docmodule(net/url)
+
+The @rhombusmodname(net/url) library provides functions for parsing and
+manipulating URLs, as well as functions for HTML form and URI component
+encoding and decoding.
+
+@table_of_contents()
+
+@section(~tag: "url"){URL Parsing}
+
+@doc(
+  class URL(
+    ~scheme: scheme :: maybe(String) = #false,
+    ~user: user :: maybe(String) = #false,
+    ~host: host :: maybe(String) = #false,
+    ~port: port :: maybe(NonnegInt) = #false,
+    ~is_path_absolute: is_path_absolute :: Boolean = #false,
+    ~path: path :: List.of(PathWithParams) = [],
+    ~query: query :: List.of(KeyValue) = [],
+    ~fragment: fragment :: maybe(String) = #false
+  )
+  class PathWithParams(
+    path :: String || Path.Dot,
+    params :: List.of(String)
+  )
+  class KeyValue(
+    key :: Symbol,
+    value :: maybe(String)
+  )
+){
+
+ A class to represent URLs as defined by
+ @hyperlink("http://www.ietf.org/rfc/rfc1939.txt"){RFC 3986}. The
```

Comment:
Seems like the wrong link: maybe <https://www.rfc-editor.org/rfc/rfc3986.html>? (Racket links to <https://www.ietf.org/rfc/rfc2396.txt>: I'm not sure if there are relevant changes.)

Reply:
Thanks! It seems that the Racket docs link to RFC 2396 at the top and RFC 3986 at the `url` struct. I think the library is as consistent with RFC 3986 as RFC 2396 (though it's difficult to be sure), so I'll change to always refer to the newer one.